packages/ra-ui-materialui/src/button/DeleteWithConfirmButton.tsx

```diff
@@ -34,6 +34,7 @@ export const DeleteWithConfirmButton = <RecordType extends RaRecord = any>(
         redirect = 'list',
         translateOptions = {},
         mutationOptions,
+        color = 'error',
```

Comment:
There are linter warnings in this file (e.g. `'styled' is defined but never used`)

You should configure your IDE to show them directly.